processor/k8sattributesprocessor/internal/kube/kube.go

```diff
@@ -169,6 +172,46 @@ type FieldExtractionRule struct {
 	From string
 }
 
+func (r *FieldExtractionRule) extractFromPodMetadata(metadata map[string]string, tags *map[string]string) {
```

Comment:
Map types are [reference types](https://stackoverflow.com/questions/28384343/accessing-a-map-using-its-reference), so you don't need to use `*map` for `tags` (unless you would want to change where the passed variable points to, but this is not the case from what I see)

Reply:
Thanks, I thought empty map would behavior differently but it is not the case.. updated.